manifest.json

```diff
@@ -138,21 +138,21 @@
               "dockerfile": "1.1/jessie/sdk",
               "os": "linux",
               "tags": {
-                "1.1.5-jessie": {}
+                "1.1.5-1.1.6-jessie": {}
               }
             },
             {
               "dockerfile": "1.1/nanoserver-sac2016/sdk",
               "os": "windows",
               "tags": {
-                "1.1.5-nanoserver-sac2016": {}
+                "1.1.5-1.1.6-nanoserver-sac2016": {}
               }
             }
           ]
         },
         {
           "sharedTags": {
-            "2.0.3": {},
+            "2.0.4": {},
```

Comment:
Seems like the shared 2.0.4 tag should be updated to 2.0.4-2.1.3.

Reply:
I updated the shared tag for the build images (below). This particular line is for the runtime-only image, so I think just "2.0.4" is ok here.

Reply:
yes - I realize now I was looking in the runtime section 😄